pom.xml

```diff
@@ -189,6 +189,16 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jackson2-api</artifactId>
         </dependency>
+		<dependency>
```

Comment:
reformat these changes so it uses spaces for indentation like the rest of the file